packages/cli/lib/dataset.js

```diff
@@ -21,94 +21,115 @@ const dataset = async (cmdObj) => {
 
     const newJson = {
       info: {
-        description: "dataset generated by pipcook",
-        url: "http:\/\/mscoco.org",
-        version: "1.0",
+        description: 'dataset generated by pipcook',
+        url: 'http:\/\/mscoco.org',
+        version: '1.0',
         year: 2014,
-        contributor: "Microsoft COCO group",
-        date_created: "2015-01-27 09:11:52.357475"
+        contributor: 'Microsoft COCO group',
+        date_created: '2015-01-27 09:11:52.357475'
       },
       images: [],
       licenses: [
         {
-          url: "http:\/\/creativecommons.org\/licenses\/by-nc-sa\/2.0\/",
+          url: 'http:\/\/creativecommons.org\/licenses\/by-nc-sa\/2.0\/',
           id: 1,
-          name: "Attribution-NonCommercial-ShareAlike License"
+          name: 'Attribution-NonCommercial-ShareAlike License'
         }
       ],
       annotations: [],
       categories: []
     };
 
-    let imageNum = 1;
-    let annotationNum = 1;
     const categoryMap = {};
-    let categoryNum = 1;
+    const imageFilesPromiseArr = [];
     for (let i = 0; i < imageDirs.length; i++) {
       const imageDir = imageDirs[i];
       try {
         fse.lstatSync(imageDir).isDirectory();
         let dirName = imageDir.split(path.sep);
         dirName = dirName[dirName.length - 1];
-        if (!fse.pathExistsSync(path.join(process.cwd(), 'annotations', dirName + '.json'))) {
+        if (!fse.pathExistsSync(
+          path.join(process.cwd(),
+            'annotations',
+            `${dirName}.json`)
+        )) {
           throw new Error('no corresponding annotation');
         }
-        const json = require(path.join(process.cwd(), 'annotations', dirName + '.json'));
+        const json = require(path.join(process.cwd(), 'annotations', `${dirName}.json`)); // eslint-disable-line
 
-        json.images.forEach((image) => {
+        json.images.forEach((image, index) => {
           newJson.images.push({
             license: 1,
             file_name: image.file_name,
             coco_url: image.coco_url,
             height: image.height,
             width: image.width,
-            id: imageNum
+            id: index + 1
           });
-          imageNum++;
         });
```

Comment:
I noticed that the strategy of ids for images, categories, and annotations has changed, each iteration of the image directories resets id to 0, please confirm if it works : )

Reply:
This PR is to change the coding style, may I ask you to keep it just like before?